### DISCUSSION TITLE
Horizon Dashboard Customization Docs

### DISCUSSION THREAD

**Discussion by Rowlando13** [laravel/framework#47594] [Ideas]
The laravel horizon dashboard is very nice and I appreciate the hard work specifically @driesvints @themsaid  @taylorotwell . 
It seems like customizing some small pieces could be very useful but it is hard to get started since there are no internals documentation. If some one could get me started like with some suggestions about the best way to do this, for example alter the vue js components, rebuild the assets, ... or a pathway to have certain common ones added as config options in horizon.php, I would be happy to make the PR. An example of one that appeals to me, is on linux using the pretty hostname instead of the static hostname. It would be nice because the pretty hostname does not affect things like dns resolution.

**Comment by henzeb** [comment]
How about https://github.com/VincentBean/horizon-extended-dashboard?

**Reply by Rowlando13** [reply]
Thanks. Seems like a might be a solution. Not much documentation. Will have to check it out.
